.github/workflows/build-spack.yml

```diff
@@ -8,33 +8,18 @@ jobs:
   build_spack:
     name: build_spack
     runs-on: ubuntu-latest
-    timeout-minutes: 5
-    container: benjaminrueth/ci-spack-pyprecice-deps-1804
+    timeout-minutes: 15
+    container: precice/ci-spack-pyprecice-deps-1804
     defaults:
       run:
         shell: "bash --login -eo pipefail {0}"
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
-      - name: Install pip3
+      - name: Move Package Script
         run: |
-          curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
-          python3 get-pip.py
-      - name: Install jinja2
-        run: pip3 install --user jinja2        
-      - name: Extract branch name  # from https://stackoverflow.com/a/58035262/5158031
-        shell: bash
-        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
-        id: extract_branch
-      - name: Specify Package Script
+          cp spack/repo/packages/py-pyprecice/package.py /sources/spack/repo/packages/py-pyprecice/
+          cp spack/repo/packages/py-pyprecice/*.patch /sources/spack/repo/packages/py-pyprecice/
+      - name: Try to build py-pyprecice with spack and test it
         run: |
-          python3 spack/jinja-instantiate.py --branch ${{ steps.extract_branch.outputs.branch }} > /opt/spack/var/spack/repos/builtin/packages/py-pyprecice/package.py
```

Comment:
Do we (still) need the explicit `openssh` dependency? I never had to add or compile it on any of the computers that I have used the package on.

Reply:
From my point of view: yes. I added it to solve a runtime dependency of OpenMPI. See https://github.com/precice/python-bindings/pull/76. @ajaust feel free to mark this conversation as resolved, if this is satisfactory.

Reply:
I just realized that `openssh` was removed in d2216f6ff9e74635c93d09a04313e58f8cf8d06a and reintroduced it in 5d725fdb09108ff8acb153229a647a033a9449d0.